### PR TITLE
scaffold crd samples

### PR DESCRIPTION
**Pull request by droot**

### CODE REVIEW

File: pkg/scaffold/resource/crd_sample.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"fmt"
+	"path/filepath"
+
```
Comment (mengqiy) [inline pkg/scaffold/resource/crd_sample.go]: nit: remove this newline.

File: pkg/scaffold/resource/crd_sample.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+)
+
+var _ input.File = &CRD{}
+
+// CRDSample scaffolds a manifest for CRD sample.
+type CRDSample struct {
+	input.Input
+
+	// Scope is Namespaced or Cluster
+	Scope string
```
Comment (mengqiy) [inline pkg/scaffold/resource/crd_sample.go]: Creating a type for Scope instead of plain string?

File: pkg/scaffold/resource/crd_sample.go (hunk 1 above)
Comment (droot) [reply pkg/scaffold/resource/crd_sample.go]: Done.

File: pkg/scaffold/resource/crd_sample.go (hunk 2 above)
Comment (droot) [reply pkg/scaffold/resource/crd_sample.go]: removed scope because we aren't using it in this case.

**Pull request merged**